src/components/com_tjcertificate/media/js/certificateImage.js

```diff
@@ -71,16 +71,27 @@ var certificateImage = {
     },
 
     copyUrl: function(element) {
-		var inputDump = document.createElement('input'),
-		hrefText = window.location.href;
-		document.body.appendChild(inputDump);
-		inputDump.value = hrefText;
-		inputDump.select();
-		document.execCommand('copy');
-		document.body.removeChild(inputDump);
+        var inputDump = document.createElement('input'),
+        hrefText = window.location.href;
+        document.body.appendChild(inputDump);
 
-		setTimeout(function() {
-			jQuery("#copyurl").popover("hide");
-		}, 1000);
-	}
+        if (element === undefined)
+        {
+            element = "#copyurl";
+        }
+        else
+        {
+            hrefText = jQuery(element).attr('data-alt-url');

```

Comment:
Can you add the same attribute in the front-end also and make the code look more cleaner here without any if else @divyachaudhari